Brewfile

```diff
@@ -0,0 +1,14 @@
+# Using this file:
+# `brew bundle --no-lock` in the root directory of the repository.
+
+# Taps (data sources)
+tap "homebrew/cask-versions" # Access alternative versions for homebrew casks
+
+# terminal programs
+brew "rbenv" unless system "rbenv", "--version"
+brew "rbenv-bundler" unless system "which", "rbenv-bundler"
+brew "ruby-build" unless system "ruby-build", "--version"
+brew "vale" unless system "vale", "--version"
```

Comment:
The Brewfile should also facilitate developers so that there's one set of dependencies required for the code base. A big example would be onboarding new joiners, right?. With this in mind, what would be the reason you've avoided bringing in tooling such as GPG (for developer commits) and AWS (for secrets management)?

Reply:
My goal was just to get our pipeline working again. It's a good idea to streamline and standardise all our tools/versions among devs though 👍